lohica-js-network/src/components/footer/FooterComponent.js

```diff
@@ -1,12 +1,17 @@
 import React from 'react';
 import {Navbar} from 'react-bootstrap';
+import './style.css';
 
 class FooterComponent extends React.Component {
     render() {
         return (
-            <Navbar fixedBottom>
+            <Navbar fixedBottom className="footer-style">
                 <p className="text-center">Some Footer</p>
             </Navbar>
+            // <div className="footer-style">
```

Comment:
Do you still need these commented lines?

Reply:
removed